app/facades/admin/pages/edit_facade.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Admin
+  module Pages
+    class EditFacade
+      attr_reader :page
+
+      ACTION = 'edit'
+
+      def initialize(page:)
+        @page = page
+      end
+
+      def sections_for_select
+        Section.widgets.map { |k, v| [k.humanize.upcase, v] }
+      end
+
+      def edit_section_link_show?(object, action)
+        object.id.present? && action == ACTION
```

Comment:
`&& action == ACTION` that's some weird stuff over here.
+ this method is called only once + it's called with string literal 'edit' :) 
so we can remove this param from the method and pretend it didn't happen